examples/src/main/scala/org/apache/spark/examples/mllib/DecisionTreeRunner.scala

```diff
@@ -354,7 +353,11 @@ object DecisionTreeRunner {
 
   /**
    * Calculates the mean squared error for regression.
+   *
+   * This is just for demo purpose. In general, don't copy this code because it is no very efficient
```

Comment:
cc @mengxr @jkbradley  

this is bad ... hopefully nobody cares about performance here.